workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -195,4 +200,20 @@
       expect(output_string).to eq(expected_output)
     end
   end
+
+  describe '#build_branch_revision' do
+    build_id = 'CozmoOne'
+    branch = 'branch'
+    revision = '111'
+
+    it 'result contains the correct url if triggering a build successfully' do
+      xml_body_str = "<build webUrl='#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=111111'/>"
+      expected_output = "#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=111111"
```

Comment:
"expected_output" has value that í the same with one in "xml_body_str", so you can create a common variable.